frontend/src/e2e-test/specs/6_mobile/confirmed-day-reservations.spec.ts

```diff
@@ -393,6 +394,17 @@ async function insertConfirmedDaysTestData() {
       end: LocalTime.of(16, 45)
     }
   }).save()
+
+  // await Fixture.attendanceReservation({
```

Comment:
was this forgotten?

Reply:
Yes, reverted these leftover draft changes!